resolving/npm-resolver/src/index.ts

```diff
@@ -258,6 +270,18 @@ async function resolveNpm (
         // ignore
       }
     }
+
+    if (opts.publishedBy) {
+      const version = pickVersionByVersionRange({
+        meta,
+        versionRange: spec.fetchSpec,
+        preferredVersionSelectors: opts.preferredVersions?.[spec.name],
+        publishedBy: opts.publishedBy,
```

Comment:
If I understand correctly, you want to see if a version is found without the date check? In that case you should not pass the date here, no?

Reply:
> If I understand correctly, you want to see if a version is found without the date check? 

Yes
> In that case you should not pass the date here, no?

In fact, publishedBy is not used inside the `pickVersionByVersionRange` function, so it doesn't matter whether it is passed or not.